hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -87,6 +87,7 @@
  */
 public class AbfsClient implements Closeable {
   public static final Logger LOG = LoggerFactory.getLogger(AbfsClient.class);
+  public static final String HUNDRED_CONTINUE_USER_AGENT = " 100-continue;";
```

Comment:
Can we replace it with something like this: HUNDRED_CONTINUE_USER_AGENT = SINGLE_WHITE_SPACE + "100-continue" + SEMI_COLON;

This way it would assert that these characters are intentionally added.

Reply:
May be we can define a constant for "100-continue" as well, I can see its being used at multiple places

Reply:
We already have a constant for 100-continue, will take this up